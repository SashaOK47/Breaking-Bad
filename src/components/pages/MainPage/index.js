import React from "react";
import style from "./styles.module.scss";
import { LazyLoadImage } from "react-lazy-load-image-component";
import img from "../../../assets/images/main-bg.webp";

const MainPage = () => {
  return (
    <main className={style.mainPage}>
      <LazyLoadImage src={img} className={style.img} alt="Main background" />
    </main>
  );
};

export default MainPage;
