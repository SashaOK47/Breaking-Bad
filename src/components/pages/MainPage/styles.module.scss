@import "../../../assets/styles/main.scss";
.mainPage {
  flex: 1 1 auto;
  .img {
    width: 100%;
    height: calc(100vh - 5rem - 5rem);
    object-fit: cover;
  }
}
