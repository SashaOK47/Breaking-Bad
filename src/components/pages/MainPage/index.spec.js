import React from "react";
import MainPage from "./index";

describe("MainPage component", () => {
  it("should render MainPage component", () => {
    const component = render(<MainPage />);
    expect(component).toMatchSnapshot();
  });
});
