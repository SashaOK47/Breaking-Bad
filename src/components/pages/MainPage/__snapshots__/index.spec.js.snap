// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component should render MainPage component 1`] = `
<main
  class="mainPage"
>
  <span
    class="img"
    style="display:inline-block"
  />
</main>
`;
